docs/python-api-intro.md

```diff
@@ -27,158 +27,156 @@ menu:
 
 # Iceberg Python API
 
-Much of the python api conforms to the java api. You can get more info about the java api [here](../api).
+Much of the python api conforms to the Java API. You can get more info about the java api [here](../api).
 
-## Catalog
-
-The Catalog interface, like java provides search and management operations for tables.
-
-To create a catalog:
+## Instal
```

Comment:
Typo: `Install`. What about "Installing" or "Installation"? "Install" is a verb so it makes a strange section heading.

Reply:
Good one, thanks!